задание 2/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace задание_2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите сторону ромба");
+            double side = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("Введите угол ромба");
+            double angle = Math.PI * double.Parse(Console.ReadLine()) / 180;
+
+            double sin = Math.Sin(angle);
+            double s = side * side * sin;
```

Comment:
Можно вычислять площадь сразу. Введение дополнительной локальной переменной излишне. Её значение используется только один раз, и она не улучшает, а скорее затрудняет понимание кода.
```suggestion
            double s = side * side * Math.Sin(angle);
```